### PR TITLE
create es.json with translated appcard section

### LINKED ISSUE #4188: [localization] Translate App Card in Spanish
### Is your proposal related to a problem?

Add translations for the App Card in Spanish

### Describe the solution you'd like

The strings need to be translated:

Screenshot 2022-09-30 at 3 43 04 PM

**Here's the guide to contributing to [localization](https://docs.tooljet.com/docs/contributing-guide/l10n)**

### PR DESCRIPTION
**Pull request by exitflynn**
fixes #4188 (if translations found satisfactory)

### CODE REVIEW

File: frontend/assets/translations/es.json [hunk 1]
```diff
@@ -0,0 +1,921 @@
...
+    },
+    "homePage":{
+        "appCard":{
+            "changeIcon":"Cambiar el ícono",
+            "addToFolder":"Añadir a la carpeta",
+            "move":"Muevete",
+            "to":"A",
+            "selectFolder":"Seleccione la carpeta",
+            "deleteApp":"Eliminar aplicación",
+            "exportApp":"Aplicación de exportación",
```
Comment (LPRegen) [inline frontend/assets/translations/es.json:348]: Here has to be `Exportar aplicación`

File: frontend/assets/translations/es.json [hunk 2]
```diff
@@ -0,0 +1,921 @@
...
+        "profile":"Profile",
+        "logout":"Logout"
+    },
+    "homePage":{
+        "appCard":{
+            "changeIcon":"Cambiar el ícono",
+            "addToFolder":"Añadir a la carpeta",
+            "move":"Muevete",
+            "to":"A",
+            "selectFolder":"Seleccione la carpeta",
```
Comment (LPRegen) [inline frontend/assets/translations/es.json:346]: I find more appropriate `Seleccionar carpeta`.

File: frontend/assets/translations/es.json [hunk 3]
```diff
@@ -0,0 +1,921 @@
...
+        },
+        "profile":"Profile",
+        "logout":"Logout"
+    },
+    "homePage":{
+        "appCard":{
+            "changeIcon":"Cambiar el ícono",
+            "addToFolder":"Añadir a la carpeta",
+            "move":"Muevete",
+            "to":"A",
```
Comment (LPRegen) [inline frontend/assets/translations/es.json:345]: Where is going to be used this? The user will be moving folders/files from one directory to another? If that is the case, I think will be more apropiate to translate it as `Hacia`.

File: frontend/assets/translations/es.json [hunk 4]
```diff
@@ -0,0 +1,921 @@
...
+            "enterNewPassword":"Enter new password"
+        },
+        "profile":"Profile",
+        "logout":"Logout"
+    },
+    "homePage":{
+        "appCard":{
+            "changeIcon":"Cambiar el ícono",
+            "addToFolder":"Añadir a la carpeta",
+            "move":"Muevete",
```
Comment (LPRegen) [inline frontend/assets/translations/es.json:344]: Hi, this has to be `Mover`, instead of `Muevete`.

**Comment by withshubh** [comment]
Closing the PR. ref: https://github.com/ToolJet/ToolJet/issues/4188#issuecomment-1283021226

**Pull request closed without merging**